libvast/vast/plugin.hpp

```diff
@@ -258,6 +263,22 @@ class transform_plugin : public virtual plugin {
   make_transform_step(const caf::settings& options) const = 0;
 };
 
+// -- store plugin ------------------------------------------------------------
+
+class store_plugin : public virtual plugin {
```

Comment:
This is missing a comment and a separate PR against docs.tenzir.com

Reply:
I've added the comment; as per our offline discussion we'll do the docs PR as a separate story at a later time.